apps/web/src/components/settings/RegistrationForm/RegisterFormSettings.tsx

```diff
@@ -151,33 +149,30 @@ export default function RegisterFormSettings({
 			personalWebsite: data.personalWebsite,
 			phoneNumber: data.phoneNumber,
 			countryOfResidence: data.countryOfResidence,
+			uploadedFile: newResumeLink,
 		});
-		// Can be optimzed to run in the modify registratuib data action later.
-		runModifyResume({ resume });
-		console.log(res);
+
+		runDeleteResume({ oldFileLink: oldResumeLink });
 	}
 
 	const { execute: runModifyRegistrationData, status: loadingState } =
 		useAction(modifyRegistrationData, {
-			onSuccess: () => {
+			onSuccess: async () => {
+				oldResumeLink = newResumeLink;
 				toast.dismiss();
 				toast.success("Data updated successfully!");
 			},
-			onError: () => {
+			onError: async () => {
+				if (newResumeLink !== c.noResumeProvidedURL)
```

Comment:
This logic here makes sense, can you just confirm that you can directly call `del` from the client?

Reply:
It actually didn't work, my bad. I changed it to call an old safe action that works now.